src/pidf_controllers/why_run_to_position_mode_is_not_a_good_idea.md

```diff
@@ -0,0 +1,57 @@
+# Why Run To Position Is Not A Good Idea
+When using a PID(F) controller to move a motor to a certain position, it is often tempting
+to use the `RUN_TO_POSITION` mode, especially since it appears in many examples.
+This motor run mode is built into the FTC SDK and is designed to move a motor to a certain encoder position autonomously.
+
+However, this mode is not ideal for a few reasons (ordered from most to least significant):
+## Locked at 20hz
+
+The `RUN_TO_POSITION` mode is locked at 20hz, meaning that the motor will only update its position every 50ms.
+
+**To understand why this is a problem:**
+
+Imagine a scenario where you are trying to move a motor to a certain position.
+If the motor is moving too fast, it will overshoot the target position and then have to correct itself.
+However, if the motor is only updating its position 20 times per second, it will not be able to correct itself quickly enough,
+leading to overshoots and even oscillations.
+* This is known as sample rate: Here's a [demo](https://www.youtube.com/watch?v=fusr9eTceEo&t=133s) of its effect
+
+### Why going Custom PID(F) is better:
+With a custom PID(F) controller, you can update the motor's position every loop. Since most FTC loops run at higher than 20hz,
+a custom solution will update the motor's position more frequently solving the problems mentioned.
+
+## Syntax Requirement and Crash Errors
+The `RUN_TO_POSITION` mode requires that you set the target position before you start the motor.
+```java
+motor.setTargetPosition(1000);
+motor.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+motor.setPower(1);
+```
+
+First, this syntax is required to be followed exactly. This means that if you want to switch between manual and automatic control,
+you will have to change the motor mode every time you switch. Programming any FSM or even any complex auto becomes quite difficult.
+
+While this seems like a minor inconvenience, if done in the wrong order, it **crashes the entire bot without outputting any useful error messages**.
+This can be a significant issue not only when debugging but also in games if it's precisely not controlled.
+
+## Customizability and Tuning
+Although the 'RUN_TO_POSITION' mode allows for some basic PID tuning, most teams don't do it and just use the default values.
+This can lead to scenarios where the motor improperly responds to disturbances or even fails to reach the target position.
+
+Moreover, the RTP mode is a simple PID controller, meaning that it doesn't have the advanced features that a custom PID(F) controller has.
```

Comment:
I'd say just remove this line
rtp is actually really weird and has some strangely fancy features, like velo PID, but cookbook doesn't need to get into those

Reply:
I see no harm in documenting that RTP runs a nested controller